tests/framework/installer/ceph_installer.go

```diff
@@ -210,6 +205,10 @@ func (h *CephInstaller) CreateK8sRookClusterWithHostPathAndDevices(namespace, sy
 		}
 	}
 
+	if err := h.WipeClusterDisks(namespace); err != nil {
```

Comment:
As discussed in slack, we could use a flag to skip the wipe in dev environments, but a separate PR is fine.